packages/react-scripts/template/README.md

```diff
@@ -346,6 +346,35 @@ Learn more about ES6 modules:
 * [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 * [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
 
+## Importing a Component using path relative to src (Absolute path)
+
+Create a file called `.env` in the root of the project, then add the following:
```

Comment:
.env is exists in `.gitignore`. So anyone that is collaborating will not have the file and need to do this in the project. I think this setting should not exists in `.env`. Can we tell them to install cross-env and do `cross-env NODE_PATH=./src react-scripts start` in .scripts in package.json instead? 

@gaearon can we make the NODE_PATH configurable in package.json? 😄  So the user can just set it once and be done with it.